### DISCUSSION TITLE
Using "==xxx==" to highlight text

### DISCUSSION THREAD

**Discussion by wbl65535** [gsantner/markor#2004] [Q&A] [answered]
What if I want to use the markdown extension syntax, such as using "==xxx==" to highlight text?

**Comment by gsantner** [comment, marked as answer]
Can you please reference the Markdown specification section where this syntax is specified? Is it something supported at GFM, CommonMark?

**Reply by stephan-dev** [reply]
TL;DR : It's not in CommonMark, but many renderers support it  (vscode, obsidian, joplin, typora...). Joplin uses markdown-it, but apparently it just converts ==foo== to `<mark>foo</mark>`. But for my workflow it would be great to have the == syntax in Markor!

Long version :
https://stackoverflow.com/questions/25104738/text-highlight-in-markdown

Also : In Joplin they use markdown-it for this , e.g. markdown-it-mark
- https://github.com/markdown-it/markdown-it-mark
- https://www.npmjs.com/package/@akiravoid/markdown-it-mark

intro : hi, great app ! Can we reopen a question from someone else ? (that would be cleaner than creating a new one, right?, but I'm not sure if commenting on a question marked as answered will do anything)
I've been missing this for so long! I had looked everywhere in Markor settings, it's the first time I have the idea to google it.

**Reply by gsantner** [reply]
@stephan-dev Flexmark is the Markdown parser used by Markor. If you find something, the project happy welcomes code contributions!
